contrib/pipelines/secadvisor/Makefile

```diff
@@ -0,0 +1,21 @@
+PROG := secadvisor
```

Comment:
This file is very similar to the one in Dummy ... can we move the make process to 'core' level and just add to SRCS all .go  files in the 'specific' folder ... so we will have single make that require parameter which is the specific 'pipeline' we want to build ?

Reply:
for now no, maybe in future I will create a pipelines/Makefile.conf which will be included by each specific Makefile